httpserver-error.lua

```diff
@@ -3,6 +3,7 @@
 -- Author: Marcos Kirsch, Gregor Hartmann
 
 return function (connection, req, args)
+   req.code = args.code

```

Comment:
I was going to remove the args eitirely in an upcoming commit and would remove the line then again.
But as I think of it that might be a breaking change to others using error.lua.
Is the interface of error.lua public?

In general for now I will follow your suggestion